packages/cloud-cognitive/src/components/Datagrid/_datagrid.scss

```diff
@@ -9,6 +9,18 @@
 @import '../../global/styles/project-settings';
 @import '../../global/styles/mixins';
 
+@import './styles/datagrid';
+@import './styles/useNestedRows';
+//@import './styles/useNestedTable';
+@import './styles/useSortableColumns';
+@import './styles/useColumnRightAlign';
+@import './styles/useStickyColumn';
+@import './styles/useActionsColumn';
+@import './styles/addons/CustomizeColumnsModal';
+@import './styles/addons/RowSizeDropdown';
+@import './styles/useSelectAllToggle';
+@import './styles/useExpandedRow';
+
```

Comment:
Can you clarify why we reimport these? It looks as though these are already imported via styles/index.

Reply:
Removed the hook scss files.